nextflow.config

```diff
@@ -20,7 +20,7 @@ params {
     diginorm_C                  = 20
```

Comment:
I'm missing the prodigal training file parameter here and in `nextflow_schema.json`.

Reply:
I added the training file parameter but I don't know If I did in the right way